docs/samples/python/transformers_recognizer.py

```diff
@@ -0,0 +1,231 @@
+
+        # If there are no specific list of entities, we will look for all of it.
+        if not entities:
+            entities = self.supported_entities
+
+        for entity in entities:
+            if entity not in self.supported_entities:
+                continue
+
+            for res in ner_results:
+                if not self.__check_label(
+                    entity, res["entity_group"], self.check_label_groups
+                ):
+                    continue
+                textual_explanation = self.DEFAULT_EXPLANATION.format(
+                    res["entity_group"]
+                )
+                explanation = self.build_transformers_explanation(
+                    round(res["score"], 2), textual_explanation
+                )
+                transformers_result = self._convert_to_recognizer_result(
+                    res, explanation
+                )
+
+                results.append(transformers_result)
+
+        return results
+
+    def _convert_to_recognizer_result(self, res, explanation) -> RecognizerResult:
+
+        entity_type = self.PRESIDIO_EQUIVALENCES.get(
+            res["entity_group"], res["entity_group"]
+        )
+        transformers_score = round(res["score"], 2)
+
+        transformers_results = RecognizerResult(
+            entity_type=entity_type,
+            start=res["start"],
+            end=res["end"],
+            score=transformers_score,
+            analysis_explanation=explanation,
+        )
+
+        return transformers_results
+
+    def build_transformers_explanation(
+        self, original_score: float, explanation: str
+    ) -> AnalysisExplanation:
+        """
+        Create explanation for why this result was detected.
+
+        :param original_score: Score given by this recognizer
+        :param explanation: Explanation string
+        :return:
+        """
+        explanation = AnalysisExplanation(
+            recognizer=self.__class__.__name__,
+            original_score=original_score,
+            textual_explanation=explanation,
+        )
+        return explanation
+
+    @staticmethod
+    def __check_label(
+        entity: str, label: str, check_label_groups: Tuple[Set, Set]
+    ) -> bool:
+        return any(
+            [entity in egrp and label in lgrp for egrp, lgrp in check_label_groups]
+        )
+
+
+if __name__ == "__main__":
+
+    from presidio_analyzer import AnalyzerEngine, RecognizerRegistry
+
+    transformers_recognizer = (
+        TransformersRecognizer()
+    )  # This would download a large (~500Mb) model on the first run
+
+    registry = RecognizerRegistry()
+    registry.add_recognizer(transformers_recognizer)
+
+    analyzer = AnalyzerEngine(registry=registry)
+
+    results = analyzer.analyze(
+        "My name is Christopher and I live in Irbid.",
+        language="en",
+        return_decision_process=True,
+    )
+    for result in results:
+        print(result)
+        print(result.analysis_explanation)
```

Comment:
Did we want to print the results?
log them?

Reply:
This is just an example, we wouldn't log them in this case

Reply:
Something the user can run locally to try it out

Reply:
Ok cool. just making sure
So approving.
Thanks